vars/nexusUpload.groovy

```diff
@@ -0,0 +1,76 @@
+import static com.sap.piper.Prerequisites.checkScript
+
+import static groovy.json.JsonOutput.toJson
+
+import com.sap.piper.PiperGoUtils
+import com.sap.piper.Utils
+
+import groovy.transform.Field
+
+@Field String METADATA_FILE = 'metadata/nexusUpload.yaml'
+@Field String STEP_NAME = getClass().getName()
+
+
+void call(Map parameters = [:]) {
```

Comment:
we should keep the groovy part consistent ...
Typical pattern, see https://github.com/SAP/jenkins-library/blob/master/vars/checkmarxExecuteScan.groovy

Reply:
Do you mean the order of defines?